fe/fe-core/src/main/java/org/apache/doris/transaction/TransactionEntry.java

```diff
@@ -163,39 +162,44 @@ public void setpLoadId(Types.PUniqueId pLoadId) {
         this.pLoadId = pLoadId;
     }
 
-    // Used for insert into select
-    public void beginTransaction(DatabaseIf database, TableIf table)
-            throws DdlException, BeginTransactionException, MetaNotFoundException, AnalysisException,
-            QuotaExceedException {
+    // Used for insert into select, return the sub_txn_id for this insert
+    public long beginTransaction(TableIf table) throws UserException {
         if (isTxnBegin()) {
```

Comment:
Does isTxnBegin indicate there is already a insert into values? If so, maybe we should change name of isTxnBegin.